src/client.js

```diff
@@ -737,6 +737,9 @@ export class StreamChat {
 				type: 'connection.recovered',
 			});
 		}
+
+		this.wsPromise = Promise.resolve();
+		this.setUserPromise = Promise.resolve();
```

Comment:
We have `await this.setUserPromise` inside queryChannels and queryUsers. If the ws connection fails the first time but then gets recovered later, we need to unblock these await by resolving there promises.